src/main/java/com/salesforce/storm/spout/dynamic/coordinator/SpoutRunner.java

```diff
@@ -85,6 +85,33 @@
      */
     private volatile boolean requestedStop = false;
 
+    /**
+     * Constructor that does not make use of a count down latch.
```

Comment:
Why? I hate having multiple constructors... if we don't need the latch, let's drop it. If we need it let's require it. Both is just messy IMO.

Reply:
thats fair, the only place that uses it is in tests, happy to rip it out of there.